docs/en/setup/backend/storages/banyandb.md

```diff
@@ -6,8 +6,7 @@ Activate BanyanDB as the storage, and set storage provider to **banyandb**.
 ```yaml
 storage:
   banyandb:
-    host: ${SW_STORAGE_BANYANDB_HOST:127.0.0.1}
-    port: ${SW_STORAGE_BANYANDB_PORT:17912}
+    endpoints: ${SW_STORAGE_BANYANDB_ENDPOINTS:127.0.0.1:17912}
```

Comment:
Let's use targets as client API says.
And with this change, e2e needs to change, as well as vocabulary docs. Because system env variable name changes.

Reply:
```suggestion
    targets: ${SW_STORAGE_BANYANDB_TARGETS:127.0.0.1:17912}
```

Ref from there, https://github.com/apache/skywalking-banyandb-java-client/blob/4872fd2b61400e0602960c36ed0406672b2a5852/src/main/java/org/apache/skywalking/banyandb/v1/client/BanyanDBClient.java#L138

Reply:
Meanwhile, please bump the BanyanDB server version to the latest. We should close to 0.5 server release.